src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -45,7 +45,8 @@ CREATE TABLE "student" (
   first_name TEXT NOT NULL,
   last_name TEXT NOT NULL,
   email TEXT NOT NULL UNIQUE,
-  assigned_case_manager_id UUID REFERENCES "user" (user_id) ON DELETE SET NULL
+  assigned_case_manager_id UUID REFERENCES "user" (user_id) ON DELETE SET NULL,
+  grade SMALLINT
```

Comment:
The newest mockups have grade on the student table and on students' pages. Although I forgot to implement it elsewhere. I can either a) remove it for now and let another issue handle it or b) write a todo comment.

Reply:
In general, try to keep a PR self contained that is focused on a specific feature/bug. This field would ideally be in the PR that adds that to the frontend/backend, but since you have already added it here, I will leave it to your discretion if you want to leave it in or remove it :)

Reply:
Got it. Sometimes I think of something and just toss it in, but I'll try to keep things more self contained from now on.